scripts/zkai_d64_external_recursion_adapter_gate.py

```
@@ -0,0 +1,858 @@
+    mutate(
+        "embedded_proof_and_verification_key_payload_relabeling",
+        "artifact_binding",
+        forge_embedded_proof_and_verification_key,
+    )
+    mutate("setup_commitment_relabeling", "setup_binding", lambda r: r["statement"].__setitem__("setup_commitment", "40" * 32))
+    mutate("proof_size_metric_smuggled", "receipt_metrics", lambda r: r.setdefault("receipt_metrics", {}).__setitem__("proof_size_bytes", 1))
+    mutate("verifier_time_metric_smuggled", "receipt_metrics", lambda r: r.setdefault("receipt_metrics", {}).__setitem__("verifier_time_ms", 0.001))
+    mutate("proof_generation_time_metric_smuggled", "receipt_metrics", lambda r: r.setdefault("receipt_metrics", {}).__setitem__("proof_generation_time_ms", 0.001))
+    mutate("statement_commitment_relabeling", "statement_commitment", lambda r: r.__setitem__("statement_commitment", "blake2b-256:" + "50" * 32), refresh=False)
+    mutate("receipt_commitment_relabeling", "receipt_commitment", lambda r: r.__setitem__("receipt_commitment", "blake2b-256:" + "60" * 32), refresh=False)
+    mutate("unknown_statement_field_added", "statement_policy", lambda r: r["statement"].__setitem__("unexpected", True))
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)